File: TaskOOP/codeOOP/person/Developer.py
from codeOOP.person.Human import Human

class Developer(Human):
    def __init__(self, firstName, sName, salary, exp, manager):
        Human.__init__(self, firstName, sName, salary, exp, manager)

    def sal(self):
        self.salary = self.satSallary()
        return float(self.salary)

    def printSallary(self):
        print("Developer", self.firstName, self.sName, "has salary", self.sal())

    def getEmployee(self):
        print ("{0} {1}, manager: {2}, experience: {3}".format(self.firstName, self.sName, self.manager.sName,
                                                        self.exp))

File: OOPJava/src/main/java/exercise1/person/Developer.java
package exercise1.person;

public class Developer extends Employee {

    public Developer(String firstName, String secondName, int salary, int experience, Manager manager) {
        super(firstName, secondName, salary, experience, manager);
    }

    @Override
    public double getSalary() {
        return super.getSalary();
    }
}
